h2o-py/h2o/model/model_base.py

```diff
@@ -341,7 +341,7 @@ def training_model_metrics(self):
         """
         return self._model_json["output"]["training_metrics"]._metric_json
     
-    def model_performance(self, test_data=None, train=False, valid=False, xval=False):
+    def model_performance(self, test_data=None, train=False, valid=False, xval=False, auc_type="none"):
```

Comment:
this parameter has only affect when test_data is provided, we should validate that if test_data is None auc_type is "none" and if not produce a warning

Reply:
Good point. Thank you @michalkurka, will do.

Reply:
Done.